src/MovieLogger.jsx

```diff
@@ -35,18 +48,35 @@ function MovieLogger() {
     return (
       <div>
         <FlixterHeader />
+
         <div className="loggingContainer">
+
+
+          {showFirstSearch &&
+          <>
           <div className="question">
             What is one movie that you really enjoyed?
           </div>
-          <MovieSearch initialRating={7.5} onMovieLogged={handleMovieLogged} />
+          <MovieSearch initialRating={7.5} onMovieLogged={()=>{handleMovieLogged(1)}} />
+          </>
+  }
```

Comment:
There are indentations issues in this PR.
All of which can be solved with the linter.